src/main/java/com/epam/aidial/core/controller/DeploymentPostController.java

```diff
@@ -69,6 +69,13 @@ public DeploymentPostController(Proxy proxy, ProxyContext context) {
         this.context = context;
         this.applicationService = proxy.getCustomApplicationService();
         this.enhancementFunctions = List.of(new CollectAttachmentsFn(proxy, context),
+                new BaseFunction<>(proxy, context) {
+                    @Override
+                    public Throwable apply(ObjectNode tree) {
+                        context.setStreamingRequest(tree.get("stream").asBoolean(false));
```

Comment:
Do you really need this flag?

Reply:
yes, we need it when constructing buffered stream for response

Reply:
fixed